includes/Hooks.php

```diff
@@ -163,7 +167,7 @@ public static function onBeforeCreateEchoEvent(
 			'group' => 'positive',
 			'section' => 'alert',
 			'canNotifyAgent' => true,
-			'presentation-model' => EchoCreateWikiPresentationModel::class,
+			'presentation-model' => \MediaWiki\Extension\CreateWiki\Notifications\EchoRequestCommentPresentationModel::class,
```

Comment:
Can we add `use` statements for this and the rest?

Reply:
Yes, I will. I just didn't because they are used only once each.

Can

https://github.com/lens0021/CreateWiki/blob/d81f4eb914170472ac81a3edb0ea5e57ffe3ceda/includes/CreateWikiOOUIForm.php#L20
```php
new \OOUI\PanelLayout();
```

be same?

Reply:
The OOUI I'm indifferent about, as it wasn't before. It's entirely necessary, but if you want to I guess it's fine.

Reply:
Ok, I'll leave OOUI as they are because I think it's simpler.